Kitodo/src/main/java/org/kitodo/production/services/data/PropertyService.java

```diff
@@ -97,12 +99,12 @@ public Long countResults(Map filters) throws DAOException {
     }
 
     private List<String> findDistinctTitles(String type) {
-        List<Property> byQuery = getByQuery("SELECT DISTINCT property.title from Property as property");
-        List<String> titles = new ArrayList<>();
+        List<Property> byQuery = getByQuery("SELECT DISTINCT property FROM Property AS property GROUP BY property.id");
```

Comment:
```suggestion
        List<Property> byQuery = getByQuery("SELECT DISTINCT property FROM Property AS property GROUP BY property.title");
```

should be a better choice